### ISSUE TITLE
Tab jitter when multiple windows have pinned tabs

### ISSUE THREAD

**Issue by irissam** [samwarring/tab_harmony#4]
1. Window 1 has 4 pinned tabs and lots of unpinned tabs (already sorted and grouped)
2. Window 2 has 1 pinned tab
3. Pick any tab from window 1. Navigate to new URL - even one in the same domain.
4. Other tabs in window 1 "jitter". Looks like they are quickly moving around, but they end up not moving anywhere.

If window 2 doesn't have any pinned tabs, then the issue does not occur.

**Comment by irissam** [comment]
The issue may lie here:
```
    getPinnedTabs: async function(windowId) {
        return chrome.tabs.query({
            pinned: true
        });
    },
```
The query to get the pinned tabs should only get those tabs for the provided window id.

**Comment by samwarring** [comment]
Fixed by d633d8f6259f985a8a8e2301b7736de3c4ee844e
